flink-connector-aws/flink-connector-dynamodb/src/main/java/org/apache/flink/connector/dynamodb/source/proxy/DynamoDbStreamsProxy.java

```diff
@@ -86,13 +88,32 @@ public GetRecordsResponse getRecords(
             }
             return getRecordsResponse;
         } catch (ExpiredIteratorException e) {
+            LOG.warn(
+                    "Received ExpiredIteratorException from GetRecords. "
+                            + "Calling GetShardIterator for shard: {} with position: {}",
+                    startingPosition,
+                    shardId);
```

Comment:
nit: This should be something like `info`, since we automatically recover here. This might happen alot of the job is backpressured.

Reply:
changed this to info